messaging/src/main/java/org/axonframework/eventhandling/TrackingEventProcessor.java

```diff
@@ -696,6 +700,12 @@ private CompletableFuture<Void> awaitTermination() {
                                     if (workerThread.isAlive()) {
                                         workerThread.interrupt();
                                         workerThread.join(workerTerminationTimeout);
+                                        if (workerThread.isAlive( )) {
```

Comment:
Nit: The indentation doesn't seem aligned with Axon Framework's default.

Reply:
Can you please provide a commit suggestion for this? I am using Eclipse and cannot use the IntelliJ settings.

Reply:
Done!